CHANGELOG.md

```diff
@@ -1,6 +1,73 @@
 # [Versions](https://mui.com/versions/)
 
-## 6.0.2
+## v6.1.0
+
+<!-- generated comparing v6.0.2..master -->
+
+_Sep 10, 2024_
+
+A big thanks to the 20 contributors who made this release possible. Here are the highlights ✨:
+
+- **Minor version bump**: This release includes a minor version bump as the `exports` field was added to the `@mui/icons-material` package to solve longstanding ESM compatibility issues (#43624). This change is backwards compatible with previously recommended setups. Bundlers which don't support the `exports` field shouldn't be affected. If you have any issues, please reach out by creating a GitHub issue.
```

Comment:
@Janpot What do you think about this summary? Should we provide more detailed guidance? Or is this enough?